pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -266,6 +323,46 @@ func injectLibConfig(pod *corev1.Pod, lang language) error {
 	for _, env := range libConfig.ToEnvs() {
 		_ = injectEnv(pod, env)
 	}
+
+	return nil
+}
+
+func setEnvVar(ctr *corev1.Container, envKey string, envValue string) error {
```

Comment:
`setEnvVar` and `setEnvVarFunc` are actually the same with an identity `envValFunc`:
```
func identityValFunc(s string) envValFunc {
  return func(string) string { return s }
}
```

Reply:
Added in 491425ab90ce7dfdc556249806debafecddab177